
Epoch 1/30


185/185 [==============================] - ETA: 0s - loss: 0.6390 - mean_absolute_error: 0.0560
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 8s 28ms/step - loss: 0.6390 - mean_absolute_error: 0.0560 - val_loss: 0.6318 - val_mean_absolute_error: 0.0310
Epoch 2/30

183/185 [============================>.] - ETA: 0s - loss: 0.6240 - mean_absolute_error: 0.0300
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 5s 26ms/step - loss: 0.6242 - mean_absolute_error: 0.0300 - val_loss: 0.6313 - val_mean_absolute_error: 0.0316
Epoch 3/30
 34/185 [====>.........................] - ETA: 2s - loss: 0.6290 - mean_absolute_error: 0.0278

162/185 [=========================>....] - ETA: 0s - loss: 0.6223 - mean_absolute_error: 0.0263
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

181/185 [============================>.] - ETA: 0s - loss: 0.6230 - mean_absolute_error: 0.0261
185/185 [==============================] - 5s 25ms/step - loss: 0.6231 - mean_absolute_error: 0.0260 - val_loss: 0.6295 - val_mean_absolute_error: 0.0237
Epoch 4/30

183/185 [============================>.] - ETA: 0s - loss: 0.6219 - mean_absolute_error: 0.0207
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 5s 25ms/step - loss: 0.6221 - mean_absolute_error: 0.0207 - val_loss: 0.6291 - val_mean_absolute_error: 0.0219
Epoch 5/30
 85/185 [============>.................] - ETA: 1s - loss: 0.6225 - mean_absolute_error: 0.0184
185/185 [==============================] - 3s 16ms/step - loss: 0.6217 - mean_absolute_error: 0.0186 - val_loss: 0.6292 - val_mean_absolute_error: 0.0235
Epoch 6/30


182/185 [============================>.] - ETA: 0s - loss: 0.6215 - mean_absolute_error: 0.0171
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 5s 26ms/step - loss: 0.6214 - mean_absolute_error: 0.0171 - val_loss: 0.6283 - val_mean_absolute_error: 0.0166
Epoch 7/30

182/185 [============================>.] - ETA: 0s - loss: 0.6215 - mean_absolute_error: 0.0171
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 4s 24ms/step - loss: 0.6214 - mean_absolute_error: 0.0170 - val_loss: 0.6282 - val_mean_absolute_error: 0.0156
Epoch 8/30
 68/185 [==========>...................] - ETA: 1s - loss: 0.6234 - mean_absolute_error: 0.0176

184/185 [============================>.] - ETA: 0s - loss: 0.6213 - mean_absolute_error: 0.0174
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 6s 30ms/step - loss: 0.6214 - mean_absolute_error: 0.0174 - val_loss: 0.6282 - val_mean_absolute_error: 0.0159
Epoch 9/30
185/185 [==============================] - 3s 18ms/step - loss: 0.6212 - mean_absolute_error: 0.0157 - val_loss: 0.6282 - val_mean_absolute_error: 0.0164
Epoch 10/30


184/185 [============================>.] - ETA: 0s - loss: 0.6212 - mean_absolute_error: 0.0162
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 5s 25ms/step - loss: 0.6212 - mean_absolute_error: 0.0162 - val_loss: 0.6282 - val_mean_absolute_error: 0.0156
Epoch 11/30

183/185 [============================>.] - ETA: 0s - loss: 0.6211 - mean_absolute_error: 0.0152
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 5s 27ms/step - loss: 0.6211 - mean_absolute_error: 0.0152 - val_loss: 0.6280 - val_mean_absolute_error: 0.0143
Epoch 12/30
 48/185 [======>.......................] - ETA: 2s - loss: 0.6217 - mean_absolute_error: 0.0168

185/185 [==============================] - 3s 17ms/step - loss: 0.6211 - mean_absolute_error: 0.0154 - val_loss: 0.6281 - val_mean_absolute_error: 0.0158
Epoch 13/30

182/185 [============================>.] - ETA: 0s - loss: 0.6209 - mean_absolute_error: 0.0150
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 4s 24ms/step - loss: 0.6211 - mean_absolute_error: 0.0150 - val_loss: 0.6280 - val_mean_absolute_error: 0.0142
Epoch 14/30
 75/185 [===========>..................] - ETA: 1s - loss: 0.6192 - mean_absolute_error: 0.0145

184/185 [============================>.] - ETA: 0s - loss: 0.6210 - mean_absolute_error: 0.0150
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 5s 28ms/step - loss: 0.6210 - mean_absolute_error: 0.0150 - val_loss: 0.6280 - val_mean_absolute_error: 0.0147
Epoch 15/30

181/185 [============================>.] - ETA: 0s - loss: 0.6207 - mean_absolute_error: 0.0148
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 4s 23ms/step - loss: 0.6210 - mean_absolute_error: 0.0148 - val_loss: 0.6280 - val_mean_absolute_error: 0.0146
Epoch 16/30
185/185 [==============================] - 3s 18ms/step - loss: 0.6210 - mean_absolute_error: 0.0144 - val_loss: 0.6280 - val_mean_absolute_error: 0.0154
Epoch 17/30


183/185 [============================>.] - ETA: 0s - loss: 0.6212 - mean_absolute_error: 0.0146
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 5s 27ms/step - loss: 0.6209 - mean_absolute_error: 0.0146 - val_loss: 0.6279 - val_mean_absolute_error: 0.0138
Epoch 18/30
185/185 [==============================] - 3s 17ms/step - loss: 0.6209 - mean_absolute_error: 0.0141 - val_loss: 0.6282 - val_mean_absolute_error: 0.0172
Epoch 19/30


182/185 [============================>.] - ETA: 0s - loss: 0.6208 - mean_absolute_error: 0.0147
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
  saving_api.save_model(
185/185 [==============================] - 4s 24ms/step - loss: 0.6209 - mean_absolute_error: 0.0147 - val_loss: 0.6278 - val_mean_absolute_error: 0.0134
Epoch 20/30
185/185 [==============================] - 3s 18ms/step - loss: 0.6209 - mean_absolute_error: 0.0143 - val_loss: 0.6285 - val_mean_absolute_error: 0.0207
Epoch 21/30

185/185 [==============================] - 3s 18ms/step - loss: 0.6209 - mean_absolute_error: 0.0144 - val_loss: 0.6280 - val_mean_absolute_error: 0.0151
Epoch 22/30
185/185 [==============================] - 3s 17ms/step - loss: 0.6209 - mean_absolute_error: 0.0140 - val_loss: 0.6282 - val_mean_absolute_error: 0.0180
Epoch 23/30


185/185 [==============================] - ETA: 0s - loss: 0.6208 - mean_absolute_error: 0.0134
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 5s 28ms/step - loss: 0.6208 - mean_absolute_error: 0.0134 - val_loss: 0.6278 - val_mean_absolute_error: 0.0133
Epoch 24/30
 46/185 [======>.......................] - ETA: 2s - loss: 0.6156 - mean_absolute_error: 0.0159

166/185 [=========================>....] - ETA: 0s - loss: 0.6207 - mean_absolute_error: 0.0143
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.
185/185 [==============================] - 4s 24ms/step - loss: 0.6209 - mean_absolute_error: 0.0141 - val_loss: 0.6278 - val_mean_absolute_error: 0.0130
Epoch 25/30
 20/185 [==>...........................] - ETA: 3s - loss: 0.6260 - mean_absolute_error: 0.0127

185/185 [==============================] - 3s 17ms/step - loss: 0.6212 - mean_absolute_error: 0.0163 - val_loss: 0.6278 - val_mean_absolute_error: 0.0136
Epoch 26/30
185/185 [==============================] - 3s 17ms/step - loss: 0.6208 - mean_absolute_error: 0.0133 - val_loss: 0.6278 - val_mean_absolute_error: 0.0136
Epoch 27/30

185/185 [==============================] - 3s 17ms/step - loss: 0.6208 - mean_absolute_error: 0.0134 - val_loss: 0.6278 - val_mean_absolute_error: 0.0143
Epoch 28/30

171/185 [==========================>...] - ETA: 0s - loss: 0.6207 - mean_absolute_error: 0.0132
/home/jean-sebastien/Documents/s7/PMC/venv/lib/python3.11/site-packages/keras/src/engine/training.py:3103: UserWarning: You are saving your model as an HDF5 file via `model.save()`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')`.

183/185 [============================>.] - ETA: 0s - loss: 0.6207 - mean_absolute_error: 0.0131
185/185 [==============================] - 5s 28ms/step - loss: 0.6207 - mean_absolute_error: 0.0131 - val_loss: 0.6278 - val_mean_absolute_error: 0.0133
Epoch 29/30
185/185 [==============================] - 3s 19ms/step - loss: 0.6207 - mean_absolute_error: 0.0129 - val_loss: 0.6278 - val_mean_absolute_error: 0.0137
Epoch 30/30


185/185 [==============================] - 3s 18ms/step - loss: 0.6207 - mean_absolute_error: 0.0131 - val_loss: 0.6280 - val_mean_absolute_error: 0.0168